### PR TITLE
This patch adds the following functionality:

### PR DESCRIPTION
**Pull request by icojb25**
* Updates the package.xml
 * Updates Allrun script for hostfile cluster runs
 * Updates CfdTools.py with helper functions to pull hostfile name and hostfile enabled usage
 * Updates the CfdAnalysis.py object to include parameters for hostfilename and hostfileenabled

This makes it possible to use a hostfile for linux runs only, for testing purposes.

Please submit merge requests to the 'master' branch.

Before finalising your merge request, please:
- Ensure that all tests pass with
```FreeCAD -t TestCfdOF```
- Increment the version number in ```package.xml```

### CODE REVIEW

**Comment by icojb25** [comment]
See #154

File: CfdOF/Solve/CfdCaseWriterFoam.py [hunk 1]
```diff
@@ -224,6 +224,9 @@ def processSystemSettings(self):
         system_settings['CasePath'] = self.case_folder
         system_settings['FoamPath'] = installation_path
         system_settings['TranslatedFoamPath'] = CfdTools.translatePath(installation_path)
+        system_settings['hostFileRequired'] = CfdTools.getSystemHostfileRequired(self.analysis_obj)
```
Comment (oliveroxtoby) [inline CfdOF/Solve/CfdCaseWriterFoam.py:227]: I'm not sure that having a separate function adds anything here...couldn't we just say
system_settings['hostFileRequred'] = self.analysis_obj.UseHostfile
and
system_Settings['hostFileName'] = self.analysis_obj.HostfileName

File: Data/Templates/case/Allrun [hunk 2]
```diff
@@ -9,8 +9,13 @@ runParallel()
     sol="${sol%.*}"
     shift
     if [ -f log."$sol" ]; then rm log."$sol"; fi
-    export OMPI_MCA_btl_vader_single_copy_mechanism=none  # Workarond for open-mpi/docker bug
+    export OMPI_MCA_btl_vader_single_copy_mechanism=none  # Workaround for open-mpi/docker bug
```
Comment (oliveroxtoby) [inline Data/Templates/case/Allrun:12]: Would you mind adding the same things in the Allmesh and Allmesh.ps1 files?

File: CfdOF/Solve/CfdCaseWriterFoam.py (hunk 1 above)
Comment (icojb25) [reply CfdOF/Solve/CfdCaseWriterFoam.py:227]: yeah sorry, thats better ...

**Pull request merged**